modules/testing/exercise2.md

```diff
@@ -0,0 +1,60 @@
+# Exercise 2
```

Comment:
Overall very nice. I think once we have the layout properly in place we can maybe format this exercise a bit better, for example as an actual table. Also, then, I don't think we need the empty "answers" part.

Reply:
Thanks! I agree!